### ISSUE TITLE
nightwatch.js should "touch" tests/logs/selenium-debug.log

### ISSUE THREAD

**Issue by martinhbramwell** [awatson1978/starrynight#72]
Selenium expects a pre-existing file : `logs/selenium-debug.log`, but boilerplate/scaffold files should never contain log files.

`.gitignore` cannot be used because then continuous integration servers will not get the file Selenium demands from GitHub.

The solution, in `nightwatch.js`, is to `touch` the file just before passing control to NightWatch.

**Comment by awatson1978** [comment]
That seems reasonable and well thought out.  Rather than `touch`, we might use `fs.writeFile()` or something similar?

**Comment by martinhbramwell** [comment]
Yeah, I didn't know the syntax used by that NodeJS library, so I referred to the operation generically.
